docs/madr/decisions/049-policy-on-namespace.md

```diff
@@ -0,0 +1,681 @@
+  name: producer-timeout
+  namespace: backend-ns
+spec:
+  to:
+    - targetRef:
+        kind: MeshService
+        name: backend
+```
+
+these resources are synced to `zone-2`:
+
+```yaml
+# zone-2, producer-timeout references backend service
+kind: MeshService
+metadata:
+  name: backend-w3r3jwi90l8dyuix
+  namespace: kuma-system
+  labels:
+    kuma.io/zone: zone-1
+    k8s.kuma.io/namespace: backend-ns
+    kuma.io/display-name: backend
+---
+kind: MeshTimeout
+metadata:
+  name: producer-timeout-qwea89uy739jdgmu
+  namespace: kuma-system
+  labels:
+    kuma.io/zone: zone-1
+    k8s.kuma.io/namespace: backend-ns
+    kuma.io/display-name: producer-timeout
+spec:
+  to:
+    - targetRef:
+        kind: MeshService
+        name: backend-w3r3jwi90l8dyuix
+        namespace: kuma-system
+```
+
+##### Policy in zone-1 references MeshHTTPRoute from zone-2
+
+As a **Backend Service Owner** I want to route all requests with `/v2` prefix to `version: v2` instances
+so that requests that need v2 version of the API ended up on correct instances.
+
+As a **Backend Service Owner** I want all requests routed to `version: v2` instances to have a 5s timeout
+so that consumers didn't wait longer than max possible processing time.
+
+As a **Frontend Service Owner** I want all requests routed to `version: v2` instances of `backend` to have 3s timeout
+so that my service meets SLO and responds withing 3s.
+
+```yaml
+# zone-1, producer route 
+apiVersion: kuma.io/v1alpha1
+kind: MeshHTTPRoute
+metadata:
+  name: backend-v2
+  namespace: backend-ns
+  labels:
+    kuma.io/zone: zone-1
+spec:
+  to:
+    - targetRef:
+        kind: MeshService
+        name: backend
+      rules:
+        - matches:
+            - path:
+                type: PathPrefix
+                value: /v2
+          default:
+            backendRefs:
+              - kind: MeshServiceSubset
+                name: backend
+                tags:
+                  version: v2
+---
+# zone-1, producer policy
+apiVersion: kuma.io/v1alpha1
+kind: MeshTimeout
+metadata:
+  name: backend-route-timeout
+  namespace: backend-ns
+  labels:
+    kuma.io/zone: zone-1
+spec:
+  to:
+    - targetRef:
+        kind: MeshHTTPRoute # doesn't work like this today, we need https://github.com/kumahq/kuma/issues/10247
+        name: backend-route
+      default:
+        http:
+          requestTimeout: 5s
+```
+
+```yaml
+# zone-2, consumer policy
+apiVersion: kuma.io/v1alpha1
+kind: MeshTimeout
+metadata:
+  name: frontend-to-backend
+  namespace: frontend-ns
+  labels:
+    kuma.io/zone: zone-2
+spec:
+  to:
+    - targetRef:
+        kind: MeshHTTPRoute # technically we can use 'name' and 'namespace' but we'd have to know hashed name of the route after syncing
+        labels:
+          kuma.io/display-name: backend-route
+          k8s.kuma.io/namespace: backend-ns
+          kuma.io/zone: zone-1
+      default:
+        http:
+          requestTimeout: 3s
+```
+
+### Referencing MeshGateway
```

Comment:
I admit I overlooked this section, feels like it belongs higher up, before example.

Reply:
Each section has it's own examples to make it easier to understand, thats why MeshGateway is later on with it's own examples